activerecord/test/models/season.rb

```diff
@@ -0,0 +1,3 @@
+class Season < ActiveRecord::Base
+  self.primary_key = nil
+end
```

Comment:
Do we need a new model for this? Surely there's at least one model without a primary key already.